### ISSUE TITLE
👤User Profile

### ISSUE THREAD

**Issue by Lakhankumawat** [Lakhankumawat/sort_it#9]
Challanges
- User Profile UI/UX
- Updating user profile / Creating user profile based on firestore data
- Managing User Data inside the app example data like UID (user id)
- Updating User Profile and Saving Data

**Comment by samitkapoor** [comment]
Hey! What details do you want to display on the Profile screen?

**Comment by Lakhankumawat** [comment]
Currently this issue will require the use of firestore Db and there are only limited members in my team to whom i can give access to , so you can work on  design and logic part but for database you can create your own on firestore or you can leave that part .

**Comment by Lakhankumawat** [comment]
@samitkapoor  Check out my other projects on flutter as well thery are also good ones have suggestions and contribute if you like to , do not forget to star them , just do it , it is inexpensive 😄

**Comment by samitkapoor** [comment]
@Lakhankumawat Yeah sure, I'd like to work on the front end of the user profile screen!

**Comment by samitkapoor** [comment]
@Lakhankumawat Yeah sure, ill take a look at your other projects and see if I can be of some help.
P.S. you can star my projects as well xD

**Comment by Lakhankumawat** [comment]
If you're not busy then we can have a short google meet and we can discuss better for profile screen , design and use cases , so i can explain you more efficiently than these chats , all up to you whether it is a big yes or small no .

**Comment by samitkapoor** [comment]
@Lakhankumawat I would love to but I have exams starting from tomorrow. So I am not able to give that much time to flutter yet. Hopefully, after exams, if you still want someone to work on the front end of the application.

**Comment by Lakhankumawat** [comment]
Great ! no issues
